sections/capoeira/adeus-povo-bom/index.qmd

```diff
@@ -0,0 +1,86 @@
+---
+title: Adeus povo bom
+subtitle: Adieu braves gens
+description: ???
+date: 2023-07-08
+date-modified: ???
+license: CC BY-SA
+author: 
+  - Z.
+  - PYM
+categories: 
+  - capoeira
+  - chant
+title-block-banner: ???
+image: ???
+image-alt: ???
+---
+
+INTRO
+
+## Le chant
+
+::: {layout-ncol=2}
+
+### Adeus povo bom 🇧🇷
+
+| Adeus povo bom, adeus
+| Adeus eu ja vou membora
+| Pelas ondas do mar, eu vim
+| Pelas ondas do mar... Eu vou membora
+
+| **Adeus povo bom, adeus**
+| **Adeus eu ja vou membora**
+| **Pelas ondas do mar, eu vim**
+| **Pelas ondas do mar... Eu vou membora**
+
+### Adieu braves gens 🇫🇷
+
+| Adieu, braves gens, adieu
+| Adieu, je m'en vais déjà
+| Par les vagues de la mer, je suis venu
+| Par les vagues de la mer... Je m'en vais
+
+| **Adieu, braves gens, adieu
+| Adieu, je m'en vais déjà
+| Par les vagues de la mer, je suis venu
+| Par les vagues de la mer... Je m'en vais**
+
+:::
+
+## Vidéo de référence
+
+{{< video https://www.youtube.com/watch?v=JTa2l3Cz2QQ >}}
+
+## Prononciation et astuces de chant
+
+Le mot "bom" se prononce avec un "o" nasal, c'est-à-dire qu'il est prononcé à la fois avec la bouche et le nez ; le "m" final ne s'entend pas !
+
+Par ailleurs, le mot "membora" est une contraction ; la forme exacte, "me embora" (voir vocabulaire), se voit ici transformée pour coller au rythme de la chanson.
+
+## Vocabulaire
+
+### Noms communs
+
+À COMPLÉTER SI TU PEUX:
+
+| Portugais     | Français                |
+| ------------- | --------------          |
+| **Adeus**     | Adieu / au revoir       |
+| **o povo**    | les gens, les personnes |
+| **as ondas ** | les vagues              |
+| **o mar**     | la mer                  |
+
+### Verbes
+
+| Portugais            | Français         | Infinitif                                 |
+| -------------------- | ---------------- | --------------------------------          |
+| **eu vim**           | je suis venu     | [vir / venir][vir]                        |
+| **eu vou me embora** | je m'en vais     | [ir-se embora / s'en aller][ir-se embora] |
+
+[vir]: https://la-conjugaison.nouvelobs.com/portugais/verbe/vir.php
+[ir-se embora]: https://www.online-translator.com/conjugation%20and%20declination/portuguese/ir-se%20embora
+
+## Culture
+
+Les esclaves enfuis ça peut être important comme thème culturel
```

Comment:
Le thème particulier de cette chanson c'est le rêve de "rentrer en Afrique" : je suis venu par la mer et je repartirai par la mer. Thème qui recoupe pas mal avec le mouvement qu'à connu toute l'Amérique et en particulier les États-Unis : le "Back to Africa Movement" c.f. https://en.wikipedia.org/wiki/Back-to-Africa_movement

Si tu as envie d'en faire un mini résumé de 3~5 lignes tu peux

Reply:
J'ai commencé mais après avoir pris quelques renseignements ça me paraît anachronique par rapport à la chanson, donc je n'ai rien mis... Hésite pas à me dire si tu veux que je le mette quand même.

Reply:
Je pense que tu peux le mettre car la chanson n'est pas si ancienne que ça (les années 30~40 je crois) c'est pas un chant si ancien que ça. Et même c'est intéressant d'aborder le sujet car le chant a été utiliser pendant les grandes périodes 30~50 et 60~70 qui sont marqué par les mouvement rastafaristes et black power entre autre. L'un comme l'autre marqué par la notion de retour physique en Afrique (comme l'a fait Mohamed Ali par exemple) ou le retour aux racine de la culture noire (mouvement des Black Panthers, redécouverte de l'islam, montée du reggae et son lien avec l'arrivée en pouvoir d'Hailé Sélassié en Éthiopie...).

Si ça t'intéresse il y a un très bon podcast sur Hailé Sélassié (de son premier nom Ras Tafari avant de prendre son nom d'empereur) sur France Info : https://www.radiofrance.fr/franceinter/podcasts/les-decolonisations-africaines/les-decolonisations-africaines-du-samedi-08-juillet-2023-5594803

(oui je profite du boulot aussi pour filer des références cool)